Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -58,14 +59,6 @@ open class ChatThreadVC: _ViewController,
     override open func setUp() {
         super.setUp()
 
-        let notificationCenter = NotificationCenter.default
```

Comment:
Why are we removing this?

Reply:
This was forgotten from a previous PR as it says in the commit message, I just noticed now only